testingScenarios

```diff
@@ -334,3 +334,57 @@ SCENARIO("game with too less rounds"){
         }
     }
 }
+
+SCENARIO("check score vector size with no spare or strike at the end"){
+    GIVEN("sccore vector of pairs") {
+        std::vector<std::pair<int,int>> v {};
+        const size_t rounds = 10;
+        WHEN ("check vector size after game") {
+            playGame(rounds);
+            THEN ("vector size should be equal 10 or 11 or 12") {
+                if (v.back().first == 10) { 
+                    REQUIRE_THAT(v.size(), Equals(12));
+                }
+                else if (v.back().first + v.back().second == 10) {
+                    REQUIRE_THAT(v.size(), Equals(12));
+                }
+                REQUIRE_THAT(v.size(), Equals(10));
+            }
+        }
+    }
+}
+
+SCENARIO("check all vector elements are digit"){
+    GIVEN("empty vecotr of pairs") {
+        std::vector<pair<int,int>> v{};
+        WHEN ("check all elements after game is played") {
+            const size_t rounds = 10;
+            playGame(rounds);
+            THEN ("all elements of score vector should be digit") {
+                auto& it = find_if(v.begin(), v.end(), isdigit);
+                REQUIRE(it == v.end());  
```

Comment:
why "==" ?

Reply:
@Nessy90 I think it's "==" because find_if return Iterator to the first element satisfying the condition or last if no such element is found.

Reply:
Maybe find_if_not algorithm will be clearer?

Reply:
> why "==" ?

Nessy, as @Vicx95 wrote, if we won't find element which is not digit (letter, white sing for example) find_if or find_if_not will return it equal vector.end()
that's why we require it==end() to be sure that none of element is not a digit
none_of, or any_of by the way would also do the job